### PR TITLE
Add Presto  inverse_weibull_cdf function

### PR DESCRIPTION
**Pull request by wills-feng**
Resolves https://github.com/facebookincubator/velox/issues/5495
This is a follow up for https://github.com/facebookincubator/velox/pull/5760

### CODE REVIEW

**Comment by aditi-pandit** [comment]
@wills-feng : Please add the fuzzer output for this function.

**Comment by wills-feng** [comment]
There is a on-going discussion about if we need to keep the float test cases in unit test as the result may be different in different time/machine. So still keep it as Draft. 
Fuzzer test result. 
```
I20240419 10:19:37.930585 27386811 ExpressionFuzzerVerifier.cpp:183] inverse_weibull_cdf 4125611 100.00% 4424083
I20240419 10:19:37.930603 27386811 ExpressionFuzzerVerifier.cpp:196] ==============================> All stats sorted by number of times the function was chosen
I20240419 10:19:37.930619 27386811 ExpressionFuzzerVerifier.cpp:198] Format: functionName numTimesSelected proportionOfTimesSelected numProcessedRows
I20240419 10:19:37.930635 27386811 ExpressionFuzzerVerifier.cpp:202] inverse_weibull_cdf 4125611 100.00% 4424083
E20240419 10:19:37.930650 27386811 ExpressionFuzzerVerifier.cpp:373] Total iterations: 181904
E20240419 10:19:37.930665 27386811 ExpressionFuzzerVerifier.cpp:374] Total failed: 9420
[==========] Running 0 tests from 0 test suites.
[==========] 0 tests from 0 test suites ran. (0 ms total)
[  PASSED  ] 0 tests.
```

**Comment by wills-feng** [comment]
@aditi-pandit I refer to[ Presto Java code base](https://github.com/prestodb/presto/blob/master/presto-main/src/test/java/com/facebook/presto/operator/scalar/TestMathFunctions.java#L1417.) to use round($f()) to avoid the different precision issue. Let me know what you think?

File: velox/functions/prestosql/tests/ProbabilityTest.cpp [hunk 1]
```diff
@@ -37,6 +37,10 @@ MATCHER(IsInf, "is Infinity") {
   return arg && std::isinf(*arg);
 }
 
+double roundToTwoDecimals(double value) {
```
Comment (aditi-pandit) [inline velox/functions/prestosql/tests/ProbabilityTest.cpp:40]: This function can be made local to the test function (inverseWeibullCdf) using it. As its not used anywhere else.

**Review by aditi-pandit — COMMENTED** [review]
Thanks @wills-feng. Have a minor comment.

**Review by aditi-pandit — APPROVED** [review]
Thanks @wills-feng

**Comment by aditi-pandit** [comment]
@Yuhta, @kagamiori, @kgpai : Please can you help with the review and merge. These are old PRs for the stats functions blocked because of the bugs in the boost library fixed by Jimmy. https://github.com/boostorg/math/pull/1007

**Review by Yuhta — APPROVED** [review]

**Review by Yuhta — APPROVED** [review]

**Pull request closed without merging**